Logic/PetBase.js

```diff
@@ -74,6 +77,15 @@ const initiatePet = (pet) => {
       pet.elementName = "dark"
       break
   }
+  for (let i = 0; i < PetProfessionBonusNames.length; i++) {
+    const valueArray = PetProfessionBonusNames[i].value.split(",")
```

Comment:
You could just use the string some method on the value array.
const petProfessionBonusValues = PetProfessionBonusNames[i].value.split(",");
if (petProfessionBonusValues.some(pet.profBonus)) {
    pet.profBonusName = PetProfessionBonusNames[i].label
}